vmod/vmod_debug.c

```diff
@@ -645,8 +645,6 @@ xyzzy_concatenate(VRT_CTX, VCL_STRANDS s)
 
 	CHECK_OBJ_NOTNULL(ctx, VRT_CTX_MAGIC);
 	r = VRT_StrandsWS(ctx->ws, NULL, s);
-	if (r != NULL && *r != '\0')
-		AN(WS_Allocated(ctx->ws, r, strlen(r) + 1));
```

Comment:
If our concern is coverage for `VRT_StrandsWS()` then we need to keep a check that the result effectively belongs to the workspace.